zebedee-cms/pom.xml

```diff
@@ -67,6 +67,14 @@
             <groupId>com.github.davidcarboni</groupId>
             <artifactId>encrypted-file-upload</artifactId>
             <version>1.0.0</version>
+            <exclusions>
```

Comment:
Seems like a sensible approach. Only question I have is are sure that this still works correctly using the newer version - just checking? Also are we considering forking this as well so we can update the dependency or make it `provided`?

Reply:
I have tested extensively locally and this does work. I also intend to fully test in the develop environment before release. There was only a small change between v1.3.0 and v1.3.2 and it was already impacted by this as the existing pom ignored 1.3.0 anyway due to the conflict.
On the second point, yes we really should fork `encrypted-file-upload` but I think this is best done as a separate 20% task.

Reply:
Good stuff. I assumed you had tested it but it wouldn't be a worthwhile review if I didn't ask 😄